site/en/codelabs/openthread-simulation/index.lab.md

```diff
@@ -1,4 +1,3 @@
----
 id: openthread-simulation
```

Comment:
Originally there were 3 dashes at the top of the page.  They might be needed on the DevSite side, can you add them back?